k8s/oidc-vault/k8s/oidc-dp-configmap.yaml

```
@@ -7,7 +7,7 @@ data:
   oidc-discovery-provider.conf: |
     log_level = "INFO"
     # TODO: Replace MY_DISCOVERY_DOMAIN with the FQDN of the Discovery Provider that you will configure in DNS
-    domain = "MY_DISCOVERY_DOMAIN"
+    domain = ["MY_DISCOVERY_DOMAIN"]
```

If you haven't already, please make sure you exercise these changes to make sure things still function. That would have caught this change in configuration key name.
```suggestion
    domains = ["MY_DISCOVERY_DOMAIN"]
```

---

yes, that's my fault